service.go

```diff
@@ -387,6 +387,19 @@ type Service interface {
 	Status() (Status, error)
 }
 
+//ConfigInfo is an optional interface which allows for certain information to be obtained.
+type ConfigInfo interface {
+	configpath() string
+}
+
+//ConfigPath returns location of the service file where applicable
+func ConfigPath(s Service) string {
```

Comment:
Remove this helper function

Reply:
If I remove it how are we going to retrieve the path?

Reply:
I'm doing it this way
```
func TestConfigPath(t *testing.T) {
	p := &program{}
	sc := &service.Config{
		Name: "go_service_test",
	}
	s, err := service.New(p, sc)
	if err != nil {
		t.Fatalf("New err: %s", err)
	}

	_, err = service.ConfigPath(s)
	if err != nil {
		t.Fatal("Failed to fetch or not implemented")
	}
}
```